src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -198,18 +202,114 @@ function transformWhere(schema, className, restWhere, options = {validate: true}
   return mongoWhere;
 }
 
+const parseObjectKeyValueToMongoObjectKeyValue = (
```

Comment:
an arrow function, they permeate `this` but I guess as it's top level it doesn't change anything